documentation/compute_environments.md

```diff
@@ -1,62 +1,70 @@
 # Compute Environments
 
-The Fornax Science Console offers a few Software environments, the most relevant of which are:
+The Fornax Science Console offers several Software environments, they are currently grouped into two containers:
 
-- **Default Astrophysics** (recommended for most use cases)
-- **High-Energy Astrophysics**
+- **Default Astrophysics** (recommended for most use cases) contain many common astronomy software, including those required to run the demo notebooks. This container image is referred to as **fornax-main**.
+- **High-Energy Astrophysics** contains high-energy software, which currently includes HEASoft. Plans exist to add CIAO, XMM-SAS and Fermitools. This container image is referred to as **fornax-hea**.
 
 
-To view the software pre-installed in each environment:
+There are two ways to view the software pre-installed in the containers:
 
-1. Open a terminal in the Fornax Science Console JupyterLab.
-2. Activate the environment you want to inspect using:
-   ```conda activate <environment-name>```
-3. Run the following command to list all installed packages:
-    ```conda list```
+- **Inside the Console**:
+The list of environment files (`.yml` for conda and requirements.txt for pip) of all the installed environments can be found in the folder `$LOCK_DIR` (defaults to /opt/lock). Each environment in the container has a corresponding file there. For instance:
```

Comment:
This would look more consistent in the rendering.
```suggestion
The list of environment files (`environment.yml` for conda and `requirements.txt` for pip) of all the installed environments can be found in the folder `$LOCK_DIR` (defaults to /opt/lock). Each environment in the container has a corresponding file there. For instance:
```